state/controller.go

```diff
@@ -142,3 +149,26 @@ func checkUpdateControllerConfig(name string) error {
 	}
 	return nil
 }
+
+// checkSpaceIsValid checks if each controller machine has addresses in the
+// the input space. If not, an error is returned.
+func (st *State) checkSpaceIsValid(configSpace string) error {
```

Comment:
I think this should give an indication of what is it valid *for*, as there can be lots of spaces that are perfectly valid for other things.

brainstorm:
checkSpaceIsValidForControllers 
checkSpaceIsAvailableToAllControllers

Reply:
I like the latter.